docs/source/esp-idf_component.rst

```diff
@@ -5,15 +5,15 @@ Arduino as a ESP-IDF component
 ESP32 Arduino lib-builder
 -------------------------
 
-For a simplified method, see `lib-builder <lib_builder>`_.
+For a simplified method, see `lib-builder <https://github.com/espressif/esp32-arduino-lib-builder>`_.
 
 Installation
 ------------
 
-.. note:: Latest Arduino Core ESP32 version is now compatible with [ESP-IDF v4.4](https://github.com/espressif/esp-idf/tree/release/v4.4). Please consider this compability when using Arduino as component in ESP-IDF.
+.. note:: Latest Arduino Core ESP32 version is now compatible with `ESP-IDF v4.4 <https://github.com/espressif/esp-idf/tree/release/v4.4>`_. Please consider this compability when using Arduino as component in ESP-IDF.
 
 - Download and install `ESP-IDF <https://github.com/espressif/esp-idf>`_.
-- Create blank idf project (from one of the examples).
+- Create blank idf project (use sample_project from /examples/get-started) or choose one of the examples.
```

Comment:
idf -> ESP-IDF. Usually, we use ESP-IDF in capital letters as our official naming rule.